### ISSUE TITLE
More performant version of Usefulness.Stream.every_nth_element/2,3

### ISSUE THREAD

**Issue by asaaki** [b-filip/usefulness#1]
In [Usefulness.Stream.every_nth_element/2,3](https://github.com/b-filip/usefulness/blob/master/lib/usefulness/stream.ex#L18-L24) you can shorten this to:

```elixir
  def every_nth_element(stream, n, skip \\ 0) 
    when n >= 0 and skip >= 0,
    do: stream |> drop(skip) |> take_every(n)
```

(I used a more concise style here.)

**Comment by b-filip** [comment]
Thanks, I didn't know about Stream.take_every.

**Comment by b-filip** [comment]
Fixed it in master branch
